apps/reference/docs/guides/platform/logs.mdx

```diff
@@ -4,78 +4,198 @@ title: Logging
 
-The log browser can be accessed in the sidebar under **Logs Explorer**. The **Logs Explorer** is for querying and aggregating project logs across products using SQL `SELECT` queries.
+### Timestamp Display and Behavior
 
-### Example
+Each log entry is stored with a `timestamp` as a `TIMESTAMP` data type. Use the appropriate [timestamp function](https://cloud.google.com/bigquery/docs/reference/standard-sql/timestamp_functions#timestamp) to utilize the `timestamp` field in a query.
 
-For example, you may enter the following into the SQL editor to query for each user's IP address:
+Raw top-level timestamp values are rendered as unix microsecond. To render the timestamps in a human-readable format, use the `DATETIME()` function to convert the unix timestamp display into an ISO-8601 timestamp.
 
 ```sql
-SELECT timestamp, h.x_real_ip
-FROM edge_logs
-  LEFT JOIN UNNEST(metadata) as m ON TRUE
-  LEFT JOIN UNNEST(m.request) AS r ON TRUE
-  LEFT JOIN UNNEST(r.headers) AS h ON TRUE
-WHERE h.x_real_ip IS NOT NULL
-```
+-- timestamp column without datetime()
+select timestamp from ....
+--  1664270180000
 
-![SELECT query example](/img/guides/platform/logs/select-query.png)
+-- timestamp column with datetime()
+select datetime(timestamp) from ....
+-- 2022-09-27T09:17:10.439Z
+```
 
-The list of supported product sources can be found under the **Sources** dropdown.
+:::info
+The rendering behaviour as unix microsecond will be removed in the future.
+:::
 
 ### Unnesting Arrays
 
-To query the metadata in the above example, you can unnest the field and "join" the unnested data. Clicking on the log row shows that log metadata is stored as an array of objects.
+Each log event stores metadata an array of objects with multiple levels, and can be seen by selecting single log events in the Logs Explorer. To query arrays, use `unnest()` on each array field and add it to the query as a join. This allows you to reference the nested objects with an alias and select their individual fields.
 
-In order to query any value that is an array, we would need to `UNNEST()` that field and add it to the query as a join, thereby allowing us to reference the nested fields within the array.
+For example, to query the edge logs without any joins:
 
-:::caution
+```sql
+select timestamp, metadata from edge_logs t
+```
 
-Large projects may run into a `Resources Exceeded` memory limit error when selecting large objects with many nested keys. To avoid this error, select individual keys separately or reduce the queried date range.
-:::
+The resulting `metadata` key is rendered as an array of objects when rendered in the Logs Explorer. This can be better visualized in this diagram, where each square in a column represents a nested object:
 
-### Functions
+![Without Unnesting](/img/guides/platform/logs/unnesting-none.png)
```

Comment:
These diagrams aren't visible in dark mode:
<img width="902" alt="Screen Shot 2022-09-27 at 12 18 30 PM" src="https://user-images.githubusercontent.com/7026076/192593671-d46ff1d8-c67b-4f10-848a-4921b77cc566.png">

Reply:
I'm assuming these diagrams are placeholder.
if we were intending to ship these, we can do better 👀🚀

Are these diagrams finalized on what they're communicating?

Reply:
we can also maybe used actual dashboard screenshots so there's context that it's happening in the dashboard

Reply:
I've switched these to use excalidraw instead.

These are conceptual diagrams for user understanding, not related to actual UI.

I've added image switching based on the theme

<img width="989" alt="Screenshot 2022-09-28 at 7 17 44 PM" src="https://user-images.githubusercontent.com/22714384/192776741-6d5148b2-f77d-4fd6-8679-cd55e1386d94.png">
<img width="1026" alt="Screenshot 2022-09-28 at 7 17 40 PM" src="https://user-images.githubusercontent.com/22714384/192776751-f134157a-343c-44fa-b36b-5205f8ea68c4.png">

<img width="1084" alt="Screenshot 2022-09-28 at 7 17 37 PM" src="https://user-images.githubusercontent.com/22714384/192776766-a82ae07f-ab06-4894-a2f1-e43af0097aff.png">
<img width="1165" alt="Screenshot 2022-09-28 at 7 17 33 PM" src="https://user-images.githubusercontent.com/22714384/192776780-907f7369-fcdb-43d9-aaee-d698c7d46511.png">